collector/nginx_plus.go

```diff
@@ -13,6 +13,10 @@ import (
 type LabelUpdater interface {
 	UpdateUpstreamServerPeerLabels(upstreamServerPeerLabels map[string][]string)
 	DeleteUpstreamServerPeerLabels(peers []string)
+	UpdateStreamUpstreamServerPeerLabels(streamUpstreamServerPeerLabels map[string][]string)
+	DeleteStreamUpstreamServerPeerLabels(peers []string)
+	UpdateStreamUpstreamServerLabels(upstreamServerPeerLabels map[string][]string)
+	DeleteStreamUpstreamServerLabels(peers []string)
 	UpdateUpstreamServerLabels(upstreamServerLabelValues map[string][]string)
 	DeleteUpstreamServerLabels(upstreamNames []string)
```

Comment:
I'd move these 2 lines above your change to keep the `UpstreamServer` stuff together.